lua/entities/sent_deployableballoons.lua

```diff
@@ -34,13 +34,13 @@ CreateConVar('sbox_maxwire_deployers', 2)
 local DmgFilter
 
 local function CreateDamageFilter()
-	if DmgFilter then return end
+	if IsValid(DmgFilter) then return end
 	local DmgFilter = ents.Create("filter_activator_name")
```

Comment:
You didn't write this code, but this avoid-double-initialization logic is wrong: by declaring `local DmgFilter` here, the `local DmgFilter` on line 34 never gets assigned to.